CHANGELOG.md

```diff
@@ -4,41 +4,64 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.9.0
+This release adds major new features since the 2.8.1 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+* Hierarchical Continuous Aggregates (aka Continuous Aggregate on top of another Continuous Aggregate)
+* Improve `time_bucket_gapfill` function allowing specifying timezone to bucket
+* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
```

Comment:
```suggestion
* Introduce fixed schedules for background jobs and the ability to check job errors.
* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
```

Reply:
**Not sure if I'm on time, but here it goes:**

This release adds major new features since the 2.8.1 release.
We deem it a moderate priority for upgrading.

This release includes the following noteworthy features:
* Hierarchical continuous aggregates (a.k.a. continuous aggregate on top of another continuous aggregate).
* Improved `time_bucket_gapfill` function allows specifying time zone to bucket.
* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
* Fixed schedules for background jobs and the ability to check job errors.
* PostgreSQL 15 support.
* Improvements in DML replication when the data node goes down.

This release also includes several bug fixes.

Reply:
Please keep the other changes (remove PostgreSQL 15 support and include number 3 instead of number 6). I'm afraid I may have missed part of the edits in the process. Apologies for that.

I just want to ensure that those that go in include the changes I made above.